source/vibe/web/internal/rest/jsclient.d

```diff
@@ -0,0 +1,106 @@
+module vibe.web.internal.rest.jsclient;
```

Comment:
I'll add an example in `vibe.web.rest` and the missing module header, as well as a doc string for `generateInterface` here.